src/PairEvaluatorColloid.h

```diff
@@ -2,27 +2,58 @@
 // Copyright (c) 2021-2024, Auburn University
 // Part of azplugins, released under the BSD 3-Clause License.
 
-/*!
- * \file PairEvaluatorColloid.h
- * \brief Defines the pair force evaluator class for colloid (integrated Lennard-Jones) potential
- */
-
 #ifndef AZPLUGINS_PAIR_EVALUATOR_COLLOID_H_
 #define AZPLUGINS_PAIR_EVALUATOR_COLLOID_H_
 
 #include "PairEvaluator.h"
 
-#ifdef NVCC
+#ifdef __HIPCC__
 #define DEVICE __device__
 #else
 #define DEVICE
 #endif
 
+namespace hoomd
+    {
 namespace azplugins
     {
 namespace detail
     {
 
+struct PairParametersColloid : public PairParameters
+    {
+#ifndef __HIPCC__
+    PairParametersColloid() : A(0), a1(0), a2(0), sigma_3(0) { }
```

Comment:
Is there a reason for `sigma_3` vs `sigma` in the names of the parameters?

Reply:
`sigma_3` (sigma^3) is used to calculate the energies and forces, whereas `sigma` is never used directly. I cleaned up the implementation a bit to make it clearer!

Reply:
thank you!